packages/orbit-components/src/Inline/helpers.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import { left } from "../utils/rtl";
+import type { Theme, ThemeProps } from "../defaultTheme";
+
+import type { SpacingToken, Position } from "./index";
+
+type Prop = "align" | "justify" | "spacing";
+
+export const formatCSS = (key: string, value: string): string => `${key}: ${value};`;
+
+const getJustify = justify => {
+  if (justify === "start") return "flex-start";
+  if (justify === "end") return "flex-end";
+
+  return "center";
+};
+
+export const normalizeSpacing = (el: SpacingToken, theme: Theme): string => {
+  const tokens = {
+    none: "",
+    XXXsmall: theme.orbit.spaceXXXSmall,
```

Comment:
All the spacing names should be in `camelCase`. Because we use also `T-Shirt` sizes it should look like `XXXSmall`. Check #2457.